diff --git a/docs/astro.config.mjs b/docs/astro.config.mjs
index bc9c7ea..35ed81e 100644
--- a/docs/astro.config.mjs
+++ b/docs/astro.config.mjs
@@ -12,2 +12,7 @@ export default defineConfig({
             title: 'Make It MMD',
+            favicon: '/favicon.png',
+            logo: {
+                replacesTitle: true,
+                src: './src/assets/logo.png',
+            },
             customCss: [
